contracts/package.json

```diff
@@ -20,7 +20,7 @@
   },
   "repository": "github:liquity/bold",
   "author": "Liquity AG",
-  "license": "GPL-3.0",
+  "license": "BSL-1.1",
```

Comment:
BSL is Boost Software License, it should be `BUSL-1.1`:
https://spdx.org/licenses/